libgalois/include/katana/PropertyGraph.h

```diff
@@ -87,11 +87,14 @@ class KATANA_EXPORT PropertyGraph {
   /// The edge EntityTypeID for each edge's most specific type
   katana::NUMAArray<EntityTypeID> edge_entity_type_id_;
 
-  // List of node and edge indexes on this graph.
+  // List of node and edge indexes on this graph. And the columns that created them to persist in json
```

Comment:
this comment isn't a doc comment but even so I think your addendum goes better above the member it describes.